src/fire.js

```diff
@@ -10,8 +10,9 @@ const config = {
 };
 
 const db = firebase.initializeApp(config);
+const facebookProvider = new firebase.auth.FacebookAuthProvider()
 
 const dashboardsRef = db.database().ref();
 export const dashesRef = dashboardsRef.child('dashboards');
 
-export default db;
+export { db, facebookProvider };
```

Comment:
In the development branch, dashesRef is exported too. Which is crucial for Redux data management in other parts of the app. Please consult the active development branch and change the code accordingly.